backend/controllers/composers.rb

```diff
@@ -1,3 +1,5 @@
+require_relative '../../../../common/mixed_content_parser'
```

Comment:
This might break for symbolic linked plugins? We could use the `ASUtils.find_base_directory`?